testsupport/kubeconfig/kubeconfig.go

```diff
@@ -0,0 +1,54 @@
+package kubeconfig
```

Comment:
I was wondering if the name is accurate, as the functionality in here seems to be modifying secrets ( actually kubeconfig in secrets) , but I cannot come up with a better name.

Reply:
Yeah, I was the same. Along with cache invalidation and off-by-one errors, naming is one of the 2 hard things in CS 😉